cloud/README.md

```diff
@@ -26,34 +26,31 @@ The following clients and Pulsar CLI tools are supported to connect to cluster t
 - pulsar-client
 - pulsar-perf
 
-## How to get token options
-
-When you use Token to connect to Pulsar cluster, you need to provide the following options:
+## How to get service URL
```

Comment:
suggest we add a full pre-condition, e.g. created a instance like `instance-Name`; created cluster like `cluster-name`; create ns like `ns-name` ....
so all the shell command will contain a command as current, and also a real command with above pre-condition

Reply:
@jiazhai done PTAL again, thanks